tests/mix/testconn.nim

```diff
@@ -0,0 +1,155 @@
+{.used.}
+
+import results, unittest2
+import std/[enumerate, sequtils]
+import ../../libp2p/protocols/[mix, ping]
+import ../../libp2p/[peerid, multiaddress, switch, builders]
+import ../../libp2p/crypto/secp
+import ../utils/async_tests
+import ../helpers
+
+proc createSwitch(
+    multiAddr: MultiAddress, libp2pPrivKey: Opt[SkPrivateKey] = Opt.none(SkPrivateKey)
+): Switch =
+  var b = SwitchBuilder
+    .new()
+    .withAddress(multiAddr)
+    .withRng(crypto.newRng())
+    .withYamux()
+    .withTcpTransport()
+    .withNoise()
```

Comment:
I think we can use [`newStandardSwitchBuilder`](https://github.com/vacp2p/nim-libp2p/blob/5f6b8e86a5c294cab9ed23afcaf496c18057d079/libp2p/builders.nim#L440)